lib/utils/createFromTemplate.js

```diff
@@ -43,18 +44,18 @@ module.exports = (templateName, destPath, options = {}) =>
         }
         resolve(
           BbPromise.all([
-            new BbPromise((gitignoreRenameResolve, gitIgnoreRenameReject) => {
+            new BbPromise(async (gitignoreRenameResolve, gitIgnoreRenameReject) => {
               const gitignorePath = join(destPath, 'gitignore');
-              exists(gitignorePath, (hasGitignore) => {
-                if (!hasGitignore) {
-                  gitignoreRenameResolve();
-                  return;
-                }
+
+              try {
+                await fs.promises.access(gitignorePath);
```

Comment:
In this scenario we need to go with regular `promise.then` syntax.

Alternatively we may first refactor this function to async/await (in other PR), and then go back to this PR and rely on async/await